### ISSUE TITLE
Update county related endpoints

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot#388]
Some updates are needed to the CRUD endpoints for counties.

The current endpoints are:
![image](https://user-images.githubusercontent.com/44526601/140051251-03df0037-50ff-4404-82f8-07b6af9fd6a1.png)

Needed updates:
- the post county endpoint should only be used for adding new counties and not for editing existing counties. It should not need an id.
- a put by id endpoint for editing counties (this logic is probably now handled in the post endpoint)
- a delete county by id endpoint

**Comment by idormenco** [comment]
will do

**Comment by izdrail** [comment]
asta e aplicatia cu care se lauda guvernul roman ca a facut blockchain votting ?

**Comment by idormenco** [comment]
> asta e aplicatia cu care se lauda guvernul roman ca a facut blockchain votting ?

Nu, această aplicație e folosita de ngo-uri pentru a monitoriza alegerile nu are treabă cu RoAEP
